proto/viam/app/v1/app.proto

```diff
@@ -975,3 +979,24 @@ message CreateKeyResponse {
   string key = 1;
   string id = 2;
 }
+
+message DeleteKeyRequest {
+  string key = 1;
+  string id = 2;
+}
+
+message DeleteKeyResponse {}
```

Comment:
noticed that the pattern seems to still declare a response type even if its empty. is that correct? very new to proto stuff

Reply:
i think that's correct!